resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+restartfile=/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/restartcounter.txt
+cur=$(head -1 $restartfile)
```

Comment:
I'm not sure the `head -1` behavior should be relied upon. `head -n 1 $restartfile` is more explicit and follows the flags that head advertises in the man page.